ansible-ipi-install/roles/shared-labs-prep/tasks/main.yml

```diff
@@ -285,6 +285,35 @@
       with_items: "{{ disable_nics }}"
   delegate_to: localhost
 
+- name: Create Manifest folder
+  block:
+    - name: Set manifest directory path
+      set_fact:
+        manifest_path: "{{ playbook_dir }}/roles/installer/files/manifests/"
+      
+    - name: Clean up any existing manifest content
```

Comment:
Why do we need to cleanup content? There might be other manifests that are needed for deployment in the same directory right?

Reply:
This is a local directory which will not contain any default manifest file, `installer` role will create a fresh folder on every build in provisioner and copy this extra manifest to it. 
To make sure the script won't apply any old manifest from the previous build, its better to clean up the folder to keep only right files in current execution. I was thinking out loud that we may be using this folder often in future for driving JetSki with different manifests.

Reply:
OK, will look at this more closely.